client/state/analytics/middleware.js

```diff
@@ -0,0 +1,40 @@
+import analytics from 'analytics';
+import invoke from 'lodash/invoke';
+import reduxAnalytics from 'redux-analytics';
+
+import {
+	BUMP_STAT,
+	RECORD_EVENT,
+	RECORD_PAGE_VIEW
+} from './constants';
+
+const eventServices = {
+	ga: ( { category, action, label, value } ) => analytics.ga.recordEvent( category, action, label, value ),
+	tracks: ( { name, properties } ) => analytics.tracks.recordEvent( name, properties )
+};
+
+const pageViewServices = {
+	ga: ( { url, title } ) => analytics.ga.recordPageView( url, title ),
+	default: ( { url, title } ) => analytics.pageView.record( url, title )
+};
+
+const statBump = ( { group, name } ) => analytics.mc.bumpStat( group, name );
+
+export const dispatcher = ( { type, payload } ) => {
+	const { service = 'default' } = payload;
+
+	switch ( type ) {
```

Comment:
Is the `type` here action type or `meta.type` ? I have trouble following how the `meta.type` gets passed around